scripts/deploy.sh

```
@@ -16,7 +16,20 @@ else
+  echo "> Grafana OTel agent 감지, 모니터링 활성화"
+  set -a
+  source "$OTEL_ENV_FILE"
+  set +a
+  JAVA_AGENT_OPTS="-javaagent:$OTEL_AGENT_JAR"
+else
+  echo "> Grafana OTel agent 미설정, 모니터링 없이 실행"
+fi
+
+nohup java $JAVA_AGENT_OPTS -jar \
   -Duser.timezone=Asia/Seoul \
   $JAR_PATH \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)